### PR TITLE
feat(cli-repl): ask for connection string on win32 doubleclick spawn …

### PR DESCRIPTION
**Pull request by addaleax**
…MONGOSH-638

### CODE REVIEW

**Comment by addaleax** [comment]
This is a skunkworks result, not planned work, so I think we at least need to also have a short discussion about whether we want to add this feature or not :)

File: packages/cli-repl/test/e2e.spec.ts [hunk 1]
```diff
@@ -1039,5 +1039,32 @@ describe('e2e', function() {
...
+
+    it('allows connecting to a host and running commands against it', async() => {
+      const connectionString = await testServer.connectionString();
+      await eventually(() => {
+        shell.assertContainsOutput('Please enter a MongoDB connection string');
+      });
+      shell.writeInputLine(connectionString);
+      await shell.waitForPrompt();
+
+      expect(await shell.executeLine('db.runCommand({whatsmyuri:1})')).to.include('you:');
```
Comment (rose-m) [inline packages/cli-repl/test/e2e.spec.ts:1062]: That is a legit command? 😄 👍

**Review by rose-m — COMMENTED** [review]
I really like the idea of having that... brings in quite a bit of convenience IMHO 👍 👏

File: packages/cli-repl/test/e2e.spec.ts (hunk 1 above)
Comment (addaleax) [reply packages/cli-repl/test/e2e.spec.ts:1062]: Yes … happy to pick another one, too :)

Comment (rose-m) [reply packages/cli-repl/test/e2e.spec.ts:1062]: No I was just surprised :D

**Comment by Anemy** [comment]
I dig it! Anything to do to get it ready to merge?

https://user-images.githubusercontent.com/1791149/128928377-be360d13-5121-4bf5-ab0c-f56673ddb3e6.mp4

**Comment by addaleax** [comment]
@Anemy Max using :SGTM: in slack for this is good enough for me :)

**Pull request merged**